FrontMen/fm-hours
# Feature/employee filter by

# Changes

Possibility to filter by name or team, based on a dropdown
Inactive changed from dropdown to toggle
Badge shows on inactive employees, next to the name or the team

## Related issues
Closes https://github.com/FrontMen/fm-hours/issues/198

## How to test
1. go to /employees
2. toggle show inactive on and off -> note inactive employees show up in the list alongside the others
3. search in the input field, example "new"
4. see list filtered by name including new
5. change from name to team see multiple employees from "New Fronteers"
6. navigate to another page without refreshing
7. come back to employees, filters persist as last used

## Screenshots
![image](https://user-images.githubusercontent.com/85111889/127490595-71274c4b-e8b0-4e1f-897c-5cbe3794aef5.png)
